indico/web/templates/forms/_person_link_widget_base.html

```diff
@@ -24,12 +24,13 @@
             <span class="alpha-order-switch i-button label icon-only icon-sort-alpha-asc {% if not use_custom_order %}active{% endif %}">
             </span>
             <div class="right">
-                <span id="add-existing-{{ field.id }}" class="i-button highlight">
-                    {% trans %}Choose{% endtrans %}
+                <span id="add-existing-{{ field.id }}" class="i-button highlight text-color"
+                      title="{% trans %}Search among Indico users{% endtrans %}">
+                    {% trans %}Search{% endtrans %}
                 </span>
                 <span id="add-new-{{ field.id }}" class="i-button"
-                      title="{% trans %}Enter a person manually{% endtrans %}">
-                    {% trans %}Enter{% endtrans %}
+                      title="{% trans %}Enter person information manually{% endtrans %}">
```

Comment:
Alright then, "person information" it is, forget about the comment :stuck_out_tongue_closed_eyes: